pipelines/working-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -7,19 +7,33 @@ include::modules/pipelines-document-attributes.adoc[]
 :context: working-with-pipelines-using-the-developer-perspective
 toc::[]
 
-You can use the *Developer* perspective of the {product-title} web console to create CI/CD Pipelines for your software delivery process while creating an application on {product-title}.
+You can use the *Developer* perspective of the {product-title} web console to create CI/CD Pipelines for your software delivery process.
+
+
+You can use the *Pipeline* or *From Git* option in the *Add* page to create Pipelines.
+The *Pipeline* option enables you to custom create a Pipeline as per your needs using the *Pipeline Builder*.
+The xref:../applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc#odc-importing-codebase-from-git-to-create-application_odc-creating-applications-using-developer-perspective[*From Git*] option enables you to create Pipelines using operator-installed Pipeline templates and resources while creating an application on {product-title}.
+
+After you create Pipelines, you can view and visually interact with your deployed Pipelines in the *Pipelines* view. You can also use the *Topology* view to interact with Pipelines created using the *From Git* option. You need to apply custom labels to a Pipeline created using the *Pipeline Builder* to see it in the *Topology* view.
 
-After you create Pipelines, you can view and visually interact with your deployed Pipelines.
 ////
 NOTE: The .NET Core runtime is not yet supported by the Pipeline templates.
 ////
 
 [Discrete]
 .Prerequisites
 
-* You have access to an {product-title} cluster and have logged in to the xref:../web_console/web-console.adoc[web console].
-* You have cluster administrator privileges to install Operators and have xref:../pipelines/installing-pipelines.adoc#installing-pipelines[installed the OpenShift Pipelines Operator].
-* You are in the xref:../web_console/odc-about-developer-perspective.adoc[Developer perspective].
+* You have access to an {product-title} cluster and have switched to the xref:../web_console/odc-about-developer-perspective.adoc[Developer perspective] in the web console.
+* You have the xref:../pipelines/installing-pipelines.adoc#installing-pipelines[OpenShift Pipelines Operator installed] in your cluster.
+* You are a cluster administrator or a user with create and edit permissions.
 * You have created a project.
 
+include::modules/op-constructing-pipelines-using-existing-tasks.adoc[leveloffset=+1]
+
+include::modules/op-starting-pipelines.adoc[leveloffset=+1]
+
 include::modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc[leveloffset=+1]
```

Comment:
Maybe flip these two sections? You discuss Pipeline Run details before Pipeline. Not an issue of course, but not the flow my mind would follow.

```suggestion
include::modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc[leveloffset=+1]

include::modules/op-starting-pipelines.adoc[leveloffset=+1]
```